crypto/sig/README.md

```diff
@@ -0,0 +1,85 @@
+mc-crypto-sig
+=============
+
+This crate provides digital signatures using the Ristretto elliptic curve points
+in a manner that meets our needs, wrapping the Schnorrkel API, and using
+types compatible with the keys crate.
+
+NOTE: The code in this crate still needs more review.
+
+The API is roughly:
+
+```
+sign_message(signing_context: &[u8], private_key: &RistrettoPrivate, message: &[u8]) -> Signature
+verify_signature(signing_context: &[u8], public_key: &RistrettoPublic, message: &[u8], sig: &Signature) -> bool
+```
+
+Putting this low-level code in its own module makes it easier to reuse
+and to audit, and hides sensitive cryptographic details from the users of the API.
+
+The main differences are:
+
+- The signature is completely deterministic, like RFC6979 and ed25519, and `sign_message`
```

Comment:
Can you provide links inline to RFC6979 and ed25519 documentation?

Reply:
:+1: